c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,50 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce) {
+  var header = '<span class="suggestion-text">' +
```

Comment:
We have already discussed it - the partials are generated only when they are used in a templateUrl in a directive. It implies creating a different suggestion-directive for each type of document. If I create a dynamic templateUrl, it will not be compiled by mako. Surely there is a workaround but I would let it to the person that actually created those functions :P